scripts/shell.sh

```diff
@@ -125,16 +128,17 @@ setup_nix() {
   }
   $NIX_USER_CHROOT_BIN "${NIX_STORE}" sh -c\
    "curl -L https://releases.nixos.org/nix/nix-${NIX_VERSION}/install > ${NIX_INSTALL_SCRIPT} && sh ${NIX_INSTALL_SCRIPT}\
+    --no-channel-add\
     --no-daemon\
     --no-modify-profile\
-    --nix-extra-conf-file ${NIX_EXTRA_CONF_PATH}"
+    --nix-extra-conf-file ${NIX_CONF_DIR}/nix.conf"
 }
 
 ensure_direnv_is_configured() {
-  local readonly project_root=$(dirname $(realpath ${__FILE__}/../docs))
+  local readonly project_root=$(dirname $(realpath ${__DIR__}/../docs))
   mkdir -p "${CACHE_ROOT}"
-  echo "[whitelist]" > ${NIX_DIRENV_CONF_PATH}
-  echo "prefix = [ \"${project_root}\" ]" >> ${NIX_DIRENV_CONF_PATH}
+  echo "[whitelist]" > ${DIRENV_CONF_PATH}
+  echo "prefix = [ \"${project_root}\" ]" >> ${DIRENV_CONF_PATH}
```

Comment:
I prefer `cat > ${DIRENV_CONF_PATH} << EOF .... EOF`, but this is also fine.

Reply:
Adjusted to your (quite sensible) tastes.